reference/5.1/Microsoft.PowerShell.Core/Get-Command.md

```diff
@@ -341,17 +341,25 @@ aliases.
 
 The acceptable values for this parameter are:
 
-- `Alias`: Gets the aliases of all PowerShell commands. For more information, see [about_Aliases](About/about_Aliases.md).
+- `Alias`: Gets the aliases of all PowerShell commands. For more information, see
+  [about_Aliases](About/about_Aliases.md).
+
 - `All`: Gets all command types. This parameter value is the equivalent of `Get-Command *`.
-- `Application`: Gets non-PowerShell files in paths listed in the **Path** environment
-  variable (`$env:path`), including .txt, .exe, and .dll files. For more information about the
-  **Path** environment variable, see [about_Environment_Variables](About/about_Environment_Variables.md).
+
+- `Application`: Gets non-PowerShell files in paths listed in the **Path** environment variable
+  (`$env:path`), including .txt, .exe, and .dll files. For more information about the **Path**
+  environment variable, see [about_Environment_Variables](About/about_Environment_Variables.md).
```

Comment:
Nit: I think we normally backtick file extensions - if backticked, maybe we can forego the leading `.`? 🤔

```suggestion
  (`$env:path`), including `.txt`, `.exe`, and `.dll` files. For more information about the
  **Path** environment variable, see
  [about_Environment_Variables](About/about_Environment_Variables.md).
```